changes.go

```diff
@@ -360,6 +366,19 @@ type AttentionSetInfo struct {
 	Reason string `json:"reason"`
 }
 
+type RecipientType string
```

Comment:
As far as I can see, other enum-type strings, we keep as stings.
We don't validate enums here.

What do you think about keeping this as string?
Or maybe: What was the rational behind adding this as its own type?

Reply:
Alternatively: Adding https://gerrit-review.googlesource.com/Documentation/user-notify.html#recipient-types as the doc link here might help.

Reply:
No strong feeling here. I just personally go for named types when it comes to enums; otherwise it's easy to typo a string and have code silently fail. You can't add the named type later on v1+, because that's technically a breaking change, which is why I added it now. I didn't add the constants because those can always be added later.

I don't mind either way though, so I'm OK with deleting if you disagree. Alternatively, happy for you to apply the suggested change and merge.

Reply:
I will go with the named type. Thanks @mvdan!